### DISCUSSION TITLE
招聘分布式存储架构师、Fil技术或运维，坐标深圳--支持分布式办公(New)

### DISCUSSION THREAD

**Discussion by goovo** [filecoin-project/community-china#206] [人才招聘]
分布式存储架构师： ------ 此岗位支持全球分布式办公
1、熟悉Ipfs/Filecoin技术架构体系；
2、具备问题分析， 概念抽象和设计，以及扎实的工程编程能力，精通 Golang，熟悉 Python / Shell 等中的任一语言；
3、 熟练掌握系统编程，包括系统调用和库函数，网络/多线程编程模型，Debug 以及 IO 性能调优工具和技能等；
4、对存储系统的架构和原理有比较深入的了解，例如 LSM Tree，文件系统原理等；
5、优秀的发现和解决问题能力，良好的沟通能力，具备团队合作精神，对新技术有学习的渴望。
6.待遇：30K-60K，优秀表现者可以额外享受奖励池分FIL。
此岗位支持北京、上海、成都、杭州、武汉、南京等人才重镇的精英们可居家办公/或创建当地办公室就近办公，也欢迎海外人才分布式办公。

Go工程师：  
1.需要熟练使用Go语言，有区块链领域项目实战经验，我们希望你有自我驱动力和较强的自学能力。
2.加分项：有IPFS开发经验。
3.待遇：15K-25K，优秀表现者可以额外享受奖励池分FIL。

Rust工程师： ------ 此岗位支持全球分布式办公
1.熟练使用Rust，掌握算法优化，对区块链了解，我们希望你有自我驱动力和较强的学习能力。
2.加分项：有IPFS开发经验。
3.待遇：15K-25K，优秀表现者可以额外享受奖励池分FIL。
此岗位支持北京、上海、成都、杭州、武汉、南京等人才重镇的精英们可居家办公/或创建当地办公室就近办公，也欢迎海外人才分布式办公。

运维工程师： ----- 此岗位 2022.5.10 已经招满了，谢谢。
1.对服务器组装，硬件架构熟悉
2.对IDC机房实施安装有实战经验
3.掌握分布式存储池的搭建和维护
4.了解区块链方面的知识
5.抗压能力强，学习能力强，善于与团队成员沟通。
6.有Filecoin运维经验。
7.待遇：15K-25K，优秀表现者可以额外享受奖励池分FIL。

请发送简历：goovo#qq.com，wx: goovolvo
公司非大小周，而是标准5天工作制，上班期间大家高效率工作，下班开心与家人享受生活。
期待能与优秀同学共事~~

原先不支持全球分布式办公的，但现在老板同意部分岗位分布式办公了。
欢迎各位优秀人才加盟，一起成就一个伟大的事业！谢谢大家！

**Comment by mkcunchu** [comment]
额，自己顶一下吧

**Reply by louxyu** [reply]
顶
